src/main/java/se/sundsvall/oepintegrator/integration/db/model/InstanceEntity.java

```diff
@@ -0,0 +1,232 @@
+package se.sundsvall.oepintegrator.integration.db.model;
+
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import java.util.List;
+import java.util.Objects;
+import org.hibernate.annotations.UuidGenerator;
+import se.sundsvall.oepintegrator.integration.db.model.enums.InstanceType;
+import se.sundsvall.oepintegrator.integration.db.model.enums.IntegrationType;
+
+@Entity
+@Table(name = "open_e_instance", indexes = {
+	@Index(name = "idx_municipality_id", columnList = "municipality_id"),
+	@Index(name = "idx_municipality_id_id", columnList = "municipality_id, id"),
+	@Index(name = "idx_integration_type", columnList = "integration_type"),
+	@Index(name = "idx_id_municipality_id", columnList = "id, municipality_id")
+})
+public class InstanceEntity {
+
+	@Id
+	@UuidGenerator
+	@Column(name = "id")
+	private String id;
+
+	@Column(name = "municipality_id")
+	private String municipalityId;
+
+	@Column(name = "integration_type")
+	@Enumerated(EnumType.STRING)
+	private IntegrationType integrationType;
+
+	@Column(name = "instance_type")
+	@Enumerated(EnumType.STRING)
```

Comment:
Add an AttributeConverter for conversion between Java enum and database String (varchar)?

(see SupportManagement for example)